paymentapi/src/test/java/fi/hel/verkkokauppa/payment/api/OnlinePaymentControllerTests.java

```diff
@@ -176,4 +178,14 @@ public void testCreateCardRenewalPayment() throws IOException, InterruptedExcept
         //VismaPayResponse responseCF2 = cancelPaymentFetcher.cancelPayment(paymentId);
         //assertEquals(responseCF2.getResult(),0, "Cancel to payment is not working.");
     }
+
+
+    @Test
+    @RunIfProfile(profile = "local")
+    public void testGetOnlinePaytrailPaymentMethods() {
+        PaymentMethodDto[] onlinePaymentMethodList = paymentPaytrailService.getOnlinePaymentMethodList("01fde0e9-82b2-4846-acc0-94291625192b", "venepaikat", "EUR");
```

Comment:
Could you add some logic to create new order and not to use hard coded order id?

Reply:
Or fetch order with api call?

Reply:
Yes sounds good, I'll add this to the test flow.